frontend/src/modules/SimulationTimeSeriesSensitivity/settings.tsx

```diff
@@ -109,95 +116,91 @@ export function settings({ moduleContext, workbenchSession, workbenchServices }:
         console.debug(`handleFrequencySelectionChange()  newFreqStr=${newFreqStr}  newFreq=${newFreq}`);
         setResamplingFrequency(newFreq);
     }
-    if (!sensitivities?.getSensitivityArr()) {
-        return <div>This is not a sensitivity ensemble</div>;
+    function handleVectorSelectChange(selection: SmartNodeSelectorSelection) {
+        setSelectedVectorName(selection.selectedNodes[0]);
```

Comment:
Add `setSelectedVectorTag(selection.selectedTags[0]);` here. See further info below regarding tags vs nodes for `VectorSelector`.